src/java/test/org/apache/zookeeper/ZKTestCase.java

```diff
@@ -51,6 +51,12 @@ public void starting(FrameworkMethod method) {
             // accidentally attempting to start multiple admin servers on the
             // same port.
             System.setProperty("zookeeper.admin.enableServer", "false");
+            // ZOOKEEPER-2693 disables all 4lw by default.
+            // Here we enable the 4lw which ZooKeeper tests depends.
+            System.setProperty("zookeeper.4lw.commands.whitelist",
+                    "ruok, envi, conf, stat, srvr, cons, dump," +
```

Comment:
ZKTestCase is base test class, covering a test scenario from this class should be avoided. May be you can add more test case in FourLetterWordsWhiteListTest to increase the coverage.